eslint.config.mjs

```diff
@@ -0,0 +1,37 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+
+import tseslint from 'typescript-eslint';
+
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+
+export default tseslint.config(
+    ...compat.extends('scality'),
+    ...tseslint.configs.recommended,
+    {
+        ignores: [
+            '**/.vscode',
+            '**/vendor/*.js',
+            '**/build',
+            '**/coverage',
+            '**/dist',
+            '**/node_modules',
+            '**/package',
+            '**/lib',
+            '**/scripts',
+            '**/eslint.config.mjs',
+        ],
```

Comment:
We can follow the same approach as here in this case: https://github.com/scality/Zenko/blob/development/2.10/tests/ctst/eslint.config.mjs#L10

It will allow us not to duplicate the gitignore list